test/drivers/test_driver_imports.py

```diff
@@ -0,0 +1,51 @@
+# import unittest.mock
+# import sys
+# import os
+# import ctypes
+import pytest
+
+
```

Comment:
I think it would be worth added a general comment that explains the basic idea here.
```
"""These tests are a very minimum smoke test on the drivers.

while we work to get a more complete test in place that includes the
hardware, these are a minimum check that the module can at least
be imported"""
```

Reply:
Done